public/admin/config.yml

```diff
@@ -7,6 +7,19 @@ media_folder: "public/images/" # Media files will be stored in the repo under st
 public_folder: "/images" # The src attribute for uploaded media will begin with /images/uploads
 
 collections:
+  - name: "broadcasts"
+    label: "Schedule"
+    folder: "schedule/broadcasts"
+    create: true
+    slug: "{{slug}}"
+    fields:
+      - { label: "Title", name: "title", widget: "string" }
+      - { label: "Subtitle", name: "subtitle", widget: "string" }
+      - { label: "Weekday", name: "weekday", widget: "select", options: ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"] }
+      - { label: "Starts", name: "startTime", widget: "string" }
```

Comment:
Shouldn't these be `datetime` widgets?

Reply:
Yes I was planning to, or even a pattern widget but couldn't play around with it with not having that on the admin panel, I am planning to do that on the secondary PR